doc/src/manual/code-loading.md

```diff
@@ -151,26 +157,35 @@ and gets `2d15fe94-a1f7-436c-a4d8-07a9a496e01c`, which indicates that in the con
 
-If applying these rules doesn't find a loadable path, the package should be considered not installed and the system should raise an error or prompt the user to install the appropriate package version.
+If any of these result in success, the path to the source code entry point will be either that result,
+the relative path from that result plus `src/Name.jl`, or fail.
+
+If no source code path is found, the lookup will fail, and the user may be prompted to install the appropriate package version or to take other corrective action (such as declaring the dependency).
 
 In the example manifest file above, to find the path of the first `Priv` package—the one with UUID `ba13f791-ae1d-465a-978b-69c3ad90f72b`—Julia looks for its stanza in the manifest file, sees that it has a `path` entry, looks at `deps/Priv` relative to the `App` project directory—let's suppose the `App` code lives in `/home/me/projects/App`—sees that `/home/me/projects/App/deps/Priv` exists and therefore loads `Priv` from there.
 
-If, on the other hand, Julia was loading the *other* `Priv` package—the one with UUID `2d15fe94-a1f7-436c-a4d8-07a9a496e01c`—it finds its stanza in the manifest, see that it does *not* have a `path` entry, but that it does have a `git-tree-sha1` entry. It then computes the `slug` for this UUID/SHA-1 pair, which is `HDkr` (the exact details of this computation aren't important, but it is consistent and deterministic). This means that the path to this `Priv` package will be `packages/Priv/HDkr/src/Priv.jl` in one of the package depots. Suppose the contents of `DEPOT_PATH` is `["/home/me/.julia", "/usr/local/julia"]`; then Julia will look at the following paths to see if they exist:
+If, on the other hand, Julia was loading the *other* `Priv` package—the one with UUID `2d15fe94-a1f7-436c-a4d8-07a9a496e01c`—it finds its stanza in the manifest, see that it does *not* have a `path` entry, but that it does have a `git-tree-sha1` entry. It then computes the `slug` for this UUID/SHA-1 pair, which is `HDkrT` (the exact details of this computation aren't important, but it is consistent and deterministic). This means that the path to this `Priv` package will be `packages/Priv/HDkrT/src/Priv.jl` in one of the package depots. Suppose the contents of `DEPOT_PATH` is `["/home/me/.julia", "/usr/local/julia"]`, then Julia will look at the following paths to see if they exist:
 
-1. `/home/me/.julia/packages/Priv/HDkr/src/Priv.jl`
-2. `/usr/local/julia/packages/Priv/HDkr/src/Priv.jl`
+1. `/home/me/.julia/packages/Priv/HDkrT`
+2. `/usr/local/julia/packages/Priv/HDkrT`
 
-Julia uses the first of these that exists to load the public `Priv` package.
+Julia uses the first of these that exists to try to load the public `Priv` package, either from the file `HDkrT`, or the file `HDKrT/src/Priv.jl`.
```

Comment:
I don't recall that we support `HDkrT` being a file and loading it as Julia source. Are you sure about this?

Reply:
This is just documenting more accurately what the code actually does. I do agree it's an odd thing to support and that we should perhaps consider dropping this functionality.

Reply:
We should definitely remove that feature rather than document it. Let's do so before anyone decides to rely on it.

Reply:
```suggestion
Julia uses the first of these that exists to try to load the public `Priv` package from the file `packages/Priv/HDKrT/src/Priv.jl` in the depot where it was found.
```